core/model/src/main/java/com/mifos/mobilewallet/model/domain/client/Client.kt

```diff
@@ -0,0 +1,45 @@
+package com.mifos.mobilewallet.model.domain.client
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class Client(
```

Comment:
Please implement using `Parcelable` annotation. This is not optimized code using convert java to kotlin.